charts/nexus-iq/tests/deployment_test.yaml

```diff
@@ -0,0 +1,84 @@
+suite: deployment
```

Comment:
@jflinchbaugh this is awesome! Just qq, it is possible to check the contents of the logs? I am thinking that maybe in the future we could dynamically check the IQ version from the logs, which will help us to fully automate the release process.

Reply:
there _is_ a capability built-in for running integration tests, but it requires that the helm chart be installed into a cluster, and then you can run the tests as pods/jobs alongside the running components. that might be the place to do such a thing, though, it requires a running cluster. maybe we can figure out a minikube instance in jenkins or use sonatype's cloudy environment at some point.